include/swift/module.modulemap

```diff
@@ -0,0 +1,157 @@
+// This is defined as one big module because there are cyclic dependencies
+// between the submodules all over the place. Breaking this up may actually
+// speed up compilation.
```

Comment:
Amen, we definitely want to get to this point! It'll help a lot with people adding cyclic dependencies between the static libraries too.